waveform-django/waveforms/models.py

```diff
@@ -1,11 +1,12 @@
 from django.db import models
 
-
 class User(models.Model):
     username = models.CharField(max_length=150, unique=True, blank=False,
         default='')
     join_date = models.DateTimeField(auto_now_add=True)
     is_admin = models.BooleanField(default=False)
+    requested_annotations = models.BooleanField(default=False)
```

Comment:
Eventually we may have to adjust this if a user wants to request annotations multiple times